assets/build_windows.xml

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<launch4jConfig>
+  <dontWrapJar>true</dontWrapJar>
+  <headerType>gui</headerType>
+  <jar>rscplus.jar</jar>
+  <outfile>C:\Users\Wolf\Downloads\rscplus-windows\rscplus.exe</outfile>
+  <errTitle></errTitle>
+  <cmdLine></cmdLine>
+  <chdir>.</chdir>
+  <priority>normal</priority>
+  <downloadUrl>https://adoptopenjdk.net/?variant=openjdk8&amp;jvmVariant=hotspot</downloadUrl>
+  <supportUrl></supportUrl>
+  <stayAlive>false</stayAlive>
+  <restartOnCrash>false</restartOnCrash>
+  <manifest></manifest>
+  <icon>C:\Users\Wolf\Downloads\rscplus-windows\jre\rscplus.ico</icon>
+  <jre>
+    <path>jre</path>
+    <bundledJre64Bit>false</bundledJre64Bit>
+    <bundledJreAsFallback>false</bundledJreAsFallback>
+    <minVersion>1.8</minVersion>
+    <maxVersion></maxVersion>
+    <jdkPreference>preferJre</jdkPreference>
+    <runtimeBits>64/32</runtimeBits>
+  </jre>
+  <versionInfo>
+    <fileVersion>1.2.0.0</fileVersion>
+    <txtFileVersion>1.2</txtFileVersion>
+    <fileDescription>Windows Launcher</fileDescription>
+    <copyright>GNU General Public License v3.0</copyright>
+    <productVersion>1.2.0.0</productVersion>
+    <txtProductVersion>1.2</txtProductVersion>
+    <productName>RSC Plus</productName>
+    <companyName></companyName>
+    <internalName>RSC Plus</internalName>
```

Comment:
We don't stylize it `RSC Plus`.

It's either rscplus, RSCPlus (where + is not allowed) or RSC+ (Preferred)

Reply:
Set to RSC+